.github/workflows/pr-checks.yml

```diff
@@ -17,46 +17,51 @@ jobs:
         node-version: [16.x]
     steps:
     - uses: actions/checkout@v2
-    - uses: ./.github/actions/setup
-      id: setup
+    - uses: Khan/actions@shared-node-cache-v0
       with:
         node-version: ${{ matrix.node-version }}
 
+    - name: Get All Changed Files
+      uses: jaredly/get-changed-files@absolute
+      id: changed
+      with:
+        format: 'json'
+
     - id: js-files
       name: Find .js changed files
-      uses: ./.github/actions/filter-files
+      uses: Khan/actions@filter-files-v0
       with:
-        changed-files: ${{ steps.setup.outputs.changed_files }}
+        changed-files: ${{ steps.changed.outputs.added_modified }}
         extensions: '.js'
 
     - name: Run Flow
       if: steps.js-files.outputs.filtered != '[]'
       run: yarn flow
 
     - id: eslint-reset
-      uses: ./.github/actions/filter-files
+      uses: Khan/actions@filter-files-v0
       name: Files that would trigger a full eslint run
       with:
-        changed-files: ${{ steps.setup.outputs.changed_files }}
+        changed-files: ${{ steps.changed.outputs.added_modified }}
```

Comment:
Does `added_modified` include `renamed_and_modified` or is this going to miss linting files that were renamed and also modified?

No change requested for this PR since this question is related to the shared action rather than this specific change, but maybe there's work to be done in the shared action?